src/components/Table/HeaderCell.tsx

```diff
@@ -49,29 +47,32 @@ export default class HeaderCell extends React.PureComponent<HeaderCellProps> {
           <span className={`${TABLE_CLASSNAME}__SortableHeaderCell__title`}>
             {column.title}
           </span>
-          {sortedInfo &&
-            sortedInfo.columnKey === colKey &&
-            sortedInfo.order && (
-              <Icon
-                name={
-                  sortedInfo.order === 'descending' ? 'caret-down' : 'caret-up'
-                }
-              />
-            )}
+          {columnSortStatus !== 'none' && (
```

Comment:
@tinkertrain Oo! Just a heads up, if you want to write this inline, I would evaluate it as a ternary, which returns `<Icon /> : null`

Reason! We've experienced weird issues in React where inline `&&` like this actually renders the string `false` in the UI. Wild. I have no idea. We experienced this in HS App.

Since then, I've been ensuring that the case doesn't happen, `null` is returned.
I remember KCD mentioning something about this as well in one of his many screencasts.

😅 

(This applies to all places this logic is used to render)

Reply:
Ohh thanks for the heads up, I'll change it